_layouts/default.html

```diff
@@ -23,6 +23,8 @@
 projects:
   - title: Binder
     url: /binder
+  - title: JupyterBook
```

Comment:
Should there be a space? This is how it is in the governance docs.

```suggestion
  - title: Jupyter Book
```

Reply:
There is no space for JH and JL, thus kept the symmetry.